tools/diagnostic-information

```diff
@@ -164,15 +164,15 @@ system_logs() {
 ipm_logs() {
     verbose "ipm logs..."
     # FIXME: check for activemq and NGINX logs
-    if [ -n "$KARAF_LOG" ] && [ -d "$KARAF_LOG" ]
+    if [ -n "${KARAF_LOG}" ] && [ -d "${KARAF_LOG}" ]
     then
-        for logfile in $(ls "$KARAF_LOG")
+        for logfile in $(ls "${KARAF_LOG}")
         do
             verbose "adding $logfile..."
-            cp -H -f "/var/log/$logfile" "${WD}/${TMPDIR}/logs/$logfile"
+            cp -H -f "${KARAF_LOG}/${logfile}" "${WD}/${TMPDIR}/logs/${logfile}"
```

Comment:
Wouldn't `logfile` value already include `${KARAF_LOG}` as a result of `ls`?

I think you meant a few lines above to have `for logfile in $(cd "${KARAF_LOG}" && ls -1)` maybe?

Reply:
well, the both obviously work. But in its current form, $logfile only holds the file name, so need to append the path ($KARAF_LOG). No need to make it more complex ;)

Reply:
Ok, rechecked... in our userland at least, `ls dirname` does list only file-basenames, not the whole paths. Must have mistook it for `find` or `ls dirname/*` results then...

But got another point to consider: several services are likely to have `error.log` and so overlap. Shouldn't we collect karaf (and others') diags logs into dedicated directories in the archive?

Reply:
Merging this as is, and addressed the directory in #337